package.json

```diff
@@ -38,16 +37,18 @@
   "devDependencies": {
     "@babel/core": "7.13.14",
     "@babel/preset-react": "7.13.13",
-    "@cypress/code-coverage": "3.9.4",
+    "@cypress/code-coverage": "^3.10.0",
+    "babel-loader": "8.0.2",
     "babel-plugin-istanbul": "6.0.0",
     "babel-plugin-transform-class-properties": "6.24.1",
     "coveralls": "3.1.0",
     "cross-env": "7.0.3",
-    "cypress": "6.9.1",
+    "cypress": "12.0.2",
     "cypress-react-unit-test": "4.17.2",
     "istanbul-lib-coverage": "3.0.0",
     "parcel-bundler": "1.12.5",
-    "start-server-and-test": "1.12.1"
+    "start-server-and-test": "1.12.1",
+    "webpack": "^5.75.0"
```

Comment:
did we need webpack?

Reply:
yeah, it's a peer dependency of webpack-preprocessor which is included by @cypress/code-coverage

https://github.com/cypress-io/cypress/blob/develop/npm/webpack-preprocessor/package.json#L75
https://github.com/cypress-io/code-coverage/blob/master/use-babelrc.js#L1